camelConnector/src/main/java/io/vantiq/extsrc/camelconn/discover/CamelRunner.java

```diff
@@ -365,94 +365,101 @@ protected ProducerTemplate findOrCreateCamelTemplate() {
     @Override
     protected void beforeStart() throws Exception {
     
-        // Set any property values provided in the configuration.  These will be used in lieu of others in the classpath
-        // This method is tolerant being passed a null value, and will do the right thing. This needs to be done
-        // before the class loader is constructed since some properties may be used as things like the URL which may
-        // be used at class-load time.
-        camelContext.getPropertiesComponent().setLocalProperties(this.camelProperties);
-    
-        if (routeBuilder == null) {
-            if (StringUtils.isEmpty(routeSpec) || StringUtils.isEmpty(routeSpecType)) {
-                log.error("No routes have been specified to run.  Either {} or {} and {} are " +
-                        "required.", "routeBuilder", "routeSpec", "routeSpecType");
-                throw new IllegalStateException("No routes have been specified to run.");
-            }
-            routeBuilder = loadRouteFromText(this.routeSpec, this.routeSpecType);
-        }
-        if (routeBasedCL == null) {
-            if (routeBuilder != null) {
-                try {
-                    routeBasedCL = constructClassLoader();
-                } catch (Exception e) {
-                    throw new RuntimeException("Unable to create route-based class loader", e);
-                }
-                originalClassLoader = camelContext.getApplicationContextClassLoader();
-                camelContext.setApplicationContextClassLoader(routeBasedCL);
-                try {
-                    camelContext.addRoutes(routeBuilder);
-                } catch (Exception e) {
-                    throw new RuntimeException("Adding routes to camel context failed", e);
```

Comment:
Strange diff, but this is mostly indenting & putting things inside a try/catch block